Dockerfile

```diff
@@ -45,10 +44,8 @@ RUN --mount=type=cache,target=/var/cache/apt --mount=type=cache,target=/var/lib/
 #ENV HTTP_PROXY=${http_proxy} \
 #    HTTPS_PROXY=${https_proxy} \
 #    NO_PROXY=${no_proxy}
-#COPY MY_ORG_ROOT.crt /usr/local/share/ca-certificates
-#RUN cd /usr/local/share/ca-certificates && \
-#    wget http://pki.my-organization.local/MY%20ORG%20ROOT.crt && \
-#    update-ca-certificates
+COPY ZScaler_Root.crt /usr/local/share/ca-certificates
+RUN update-ca-certificates
```

Comment:
Revert this back to the original commented commands... (I guess this is an oversight from local testing?)